packages/core/admin/admin/src/core/apis/content-manager.ts

```diff
@@ -105,6 +107,8 @@ class ContentManagerPlugin {
     ...DEFAULT_ACTIONS,
     ...DEFAULT_TABLE_ROW_ACTIONS,
     ...DEFAULT_HEADER_ACTIONS,
+    // TODO: also check license before adding history action
+    ...(window.strapi.future.isEnabled('history') ? [HistoryAction] : []),
```

Comment:
I think I preferred it the way you had it, much neater imo, what made you change it?

Reply:
It was to avoid needing to update the content manager tests, but I can restore it and do that